docs/userguide/main.md

```diff
@@ -162,6 +162,21 @@ This allows one to set up any of the [backend](https://opensearch.org/docs/lates
 
 *The configuration must be valid or the dashboard will fail to start.*
 
+## Configuring a basePath
+
+When using OpenSearch behind a reverse proxy you have to configure a base path. This can be achieved by setting the base path field in the configuraiton of OpenSearch Dashboards. Behind the scenes the according configuration fields are set automatically in the opensearch.yml file.
+
+```yaml
+apiVersion: opensearch.opster.io/v1
+kind: OpenSearchCluster
+...
+spec:
+  dashboards:
+    enable: true
+    basePath: "/logs"
+```
+
```

Comment:
There should be a extra info that this sets `server.rewriteBasePath: true` as some might not want to use this option.

Reply:
Maybe a few words also about what is reverse proxy?